src/components/Requirements.vue

```diff
@@ -1,26 +1,276 @@
+                <p class="col float-right text-right p-0 req-progress">({{ value.progress }}/{{ value.total }} {{ value.count }})</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- Add separator if not last req or not displaying details -->
+      <div class="separator" v-if="index < reqs.length - 1 || req.displayDetails"></div>
+    </div>
   </div>
 </template>
 
 <script>
 import Vue from 'vue';
 
-export default {}
+export default {
+  data() {
+    return {
+      reqs: [
+        {
+          name: "College Requirements",
+          type: "College",
+          count: "Credits",
+          progress: 46,
+          total: 120,
+          color: '#2BBCC6',
+          displayDetails: false,
+          todo: [
+            {
+              name: "CALS Credits",
+              count: "Credits",
+              progress: 12,
+              total: 55
+            },
+            {
+              name: "PE Credits",
+              count: "Credits",
+              progress: 1,
+              total: 2
+            }
+          ],
+          displayCompleted: true,
+          completed: [
+            {
+              name: "Quantitative Literacy",
+              count: "Credits",
+              progress: 2,
+              total: 2
+            },
+            {
+              name: "Chemistry/Physics",
+              count: "Credits",
+              progress: 1,
+              total: 2
+            }
+          ]
+        },
+        {
+          name: "Major Requirements",
+          type: "Major",
+          count: "Courses",
+          progress: 4,
+          total: 8,
+          color: '#4D53DC',
+          displayDetails: false,
+          todo: [
+            {
+              name: "CALS Credits",
+              count: "Credits",
+              progress: 12,
+              total: 55
+            },
+            {
+              name: "PE Credits",
+              count: "Credits",
+              progress: 1,
+              total: 2
+            }
+          ],
+          displayCompleted: true,
+          completed: [
+            {
+              name: "Quantitative Literacy",
+              count: "Credits",
+              progress: 2,
+              total: 2
+            },
+            {
+              name: "Chemistry/Physics",
+              count: "Credits",
+              progress: 1,
+              total: 2
+            }
+          ]
+        }
+      ]
+    }
+  },
+
+  methods: {
+    turnDetails: function(index, bool) {
+      this.reqs[index].displayDetails = bool;
+    },
+    turnCompleted: function(index, bool) {
+      this.reqs[index].displayCompleted = bool;
+    }
+  }
+}
 </script>
 
 <style scoped lang="scss">
+
+.btn {
+  padding: 0;
+}
+
 .requirements {
-  width: 25rem;
-  background-color: lightblue;
-  border: 2px black solid;
-
-  &-ayesha {
-    width: 25rem;
-    font-size: 50px;
-    color: black;
-    text-align: center;
+  width: 30rem;
```

Comment:
I thought Shane said it was 400px?

Reply:
Hmm not sure why I did that. Pretty sure it's just cause I like using rem more than px. Changed it back to specs now